conans/client/tools/system_pm.py

```diff
@@ -11,7 +11,7 @@
 
 class SystemPackageTool(object):
 
-    def __init__(self, runner=None, os_info=None, tool=None, recommends=False, output=None):
+    def __init__(self, runner=None, os_info=None, tool=None, recommends=False, output=None, conanfile=None):
 
```

Comment:
Now we can get the output from the conanfile, we may use it.

```suggestion

output = output if output else conanfile.output if conanfile else None
```

Please, check that this line works.

Reply:
> Now we can get the output from the conanfile, we may use it.

You guys ask for few changes, my record is 32 requests from my past job :sweat_smile: